### DISCUSSION TITLE
Related content

### DISCUSSION THREAD

**Discussion by trevorsaint** [ONSdigital/design-system#1283] [Feedback]
Use this place to discuss the [related content component](https://ons-design-system.netlify.com/components/related-content) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Related content component can't be used within the `block aside` in the base page template, because it currently uses `aside` itself.

**Reply by jrbarnes9** [reply]
Raised #2457
